website/static/js/fangorn.js

```diff
@@ -1009,6 +1020,34 @@ function _removeEvent (event, items, col) {
     }
 }
 
+function doCheckout(item, checker, error_message) {
+    $.ajax({
+        method: 'put',
+        url: window.contextVars.apiV2Prefix + 'files' + item.data.path + '/',
+        beforeSend: $osf.setXHRAuthorization,
+        contentType: 'application/json',
+        dataType: 'json',
+        data: JSON.stringify({
+            data: {
+                id: item.data.path.replace('/', ''),
+                type: 'files',
+                attributes: {
+                    checkout: checker
+                }
+            }
+        })
+    }).done(function(resp) {
+        if (error_message) {
```

Comment:
Why do you only reload the page if `error_message` is defined? Is the page supposed to be reloaded on any successful request?